src/main/java/mate/academy/intro/service/impl/AuthenticationServiceImpl.java

```diff
@@ -46,4 +49,10 @@ public UserResponseDto register(UserRegistrationRequestDto requestDto)
 
         return userMapper.toDto(user);
     }
+
+    @Override
+    public UserLoginResponseDto authenticate(UserLoginRequestDto requestDto) {
+        String token = jwtUtil.generateToken(requestDto.email());
+        return null;
```

Comment:
Have you checked your solution in Postman? 
```suggestion
        final Authentication authentication = manager
                .authenticate(
                        new UsernamePasswordAuthenticationToken(
                                requestDto.getEmail(), requestDto.getPassword()));
        String token = jwtUtil.generateToken(authentication.getName());
        return new UserLoginResponseDto(token);
```